lib/logstash/inputs/beats/message_listener.rb

```diff
@@ -181,21 +181,13 @@ def set_nested(hash, field_name, value)
 
     private
     def extract_target_field(hash)
-      if from_filebeat?(hash)
-        hash.delete(FILEBEAT_LOG_LINE_FIELD).to_s
-      elsif from_logstash_forwarder?(hash)
-        hash.delete(LSF_LOG_LINE_FIELD).to_s
-      end
+      hash.delete(FILEBEAT_LOG_LINE_FIELD).to_s
```

Comment:
The tests fails because expects a `nil` instead received an empty string.
I think the code to satisfy again the CI is
```suggestion
      return nil if !from_filebeat?(hash)
      hash.delete(FILEBEAT_LOG_LINE_FIELD).to_s  
```